src/shared/schema/util.ts

```diff
@@ -0,0 +1,19 @@
+/*!
```

Comment:
this code can live in `src/shared/schemas.ts`. Don't need a new file

Reply:
we probably dont need the build though, right? can `git clone` in the CI buildspec.

Reply:
Yeah, we don't need the build script anymore. We can just `git clone` directly. I'm currently re-working it